client/src/components/pages/clans/clan/components/clan-actions/index.tsx

```diff
@@ -17,9 +24,14 @@ const ClanActions: FC<IClanActionsProps> = ({ visitor, handleJoin, handleLeave,
 			</Button>
 		)}
 		{visitor.isAdmin && (
-			<Button className={clsx(ButtonClasses.red, ButtonClasses.filled)} onClick={handleEdit}>
-				Edit
-			</Button>
+			<>
+				<Button className={clsx(ButtonClasses.red, ButtonClasses.filled)} onClick={handleEdit}>
+					Edit
+				</Button>
+				<Button className={clsx(ButtonClasses.red, ButtonClasses.filled)} onClick={handleDelete}>
+					Delete
+				</Button>
```

Comment:
Delete clan button already exists in edit clan modal. Remove it from modal / clan-actions component.